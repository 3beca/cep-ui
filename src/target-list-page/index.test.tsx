import * as React from 'react';
import {render} from '@testing-library/react';

import TargetListPage from './index';

test('Snapshot TargetListPage', () => {
    const {container, getByLabelText} = render(<TargetListPage/>);

    getByLabelText(/target list page/i);
    expect(container).toMatchSnapshot();
});
