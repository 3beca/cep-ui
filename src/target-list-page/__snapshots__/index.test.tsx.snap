// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot TargetListPage 1`] = `
<div>
  <div
    aria-label="target list page"
    class="makeStyles-root-1"
  >
    TargetList Page
  </div>
</div>
`;
